universite-du-quebec-a-montreal.csl

```diff
@@ -52,9 +52,6 @@
         <multiple>éditions</multiple>
       </term>
       <term name="edition" form="short">éd.</term>
-      <term name="ordinal">&lt;sup&gt;e&lt;/sup&gt;</term>
-      <term name="ordinal-01" gender-form="feminine" match="whole-number">&lt;sup&gt;re&lt;/sup&gt;</term>
-      <term name="ordinal-01" gender-form="masculine" match="whole-number">&lt;sup&gt;er&lt;/sup&gt;</term>
```

Comment:
@jjrondeau, @adam3smith already removed these lines from your submission, but we don't allow HTML tags like this for superscript.

The best solution we currently have is to use superscript characters. See the French CSL locale for an example: https://github.com/citation-style-language/locales/blob/b8c370c8a978790d2aeefa302f05f3bfb1478e75/locales-fr-FR.xml#L74

Reply:
Thank you for the example. I will use it.